### DISCUSSION TITLE
overriding org.freedesktop.DBus.Properties GetAll method

### DISCUSSION THREAD

**Discussion by 12-3-8-s9b9o9j9t** [Kistler-Group/sdbus-cpp#266] [General]
Hello, I would like to override the `GetAll` method of the `org.freedesktop.DBus.Properties` interface, so that it ignore properties when it is empty vectors or maps, is that possible ?

**Comment by sangelovic** [comment]
No, AFAIK, the underlying library -- libsystemd -- handles and implements that method itself; it doesn't bubble up to be handled by sdbus-c++. Hence, I'm not aware of any way of overriding this at the level of sdbus-c++.

**Reply by 12-3-8-s9b9o9j9t** [reply]
Ok, thank you for the answer.
